### PR TITLE
Add stand tests

### PR DESCRIPTION
**Pull request by awirb**
* Add file location where DEMs should be stored when generated with generateTopo.py 
* Add run script that generates the folder structure, topography, release area for given settings and saves the topo and release area at the correct location 
* Add simple release area generation

### CODE REVIEW

File: avaframe/in3Utils/getReleaseArea.py [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+    rel_name = cfgFile['rel_name']
+
+    # Add vertical coordinate
+    z = np.zeros((lenp, 1))
+    p_mat = np.matrix(np.append(xyPoints, z, axis=1))
+
+    # Save elevation data to .asc file and add header lines
+    with open(os.path.join(outDir, 'release_%d%s.nxyz' % (rel_no, DEM_type)), 'w') as f:
+        f.write('name=%s\n' % (rel_name))
+        f.write('ah=%.2f\n' % (rel_h))
```
Comment (fso42) [inline avaframe/in3Utils/getReleaseArea.py:161]: add a 
#TODO: change to d0
...

File: avaframe/in3Utils/getReleaseArea.py [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+def makeOutputDir(avalancheDir):
+    """ Make output directory """
+
+    outDir = os.path.join(avalancheDir, 'Inputs', 'REL')
+    if os.path.isdir(outDir):
+        log.warning('Be careful %s already existed - new data is added' % (outDir))
+    else:
+        os.makedirs(outDir)
+
+    return outDir
```
Comment (fso42) [inline avaframe/in3Utils/getReleaseArea.py:183]: See question on makeOutputDir for topography..
Also: could this be solved with one makeOutputDir function?

File: avaframe/in3Utils/getReleaseAreaCfg.ini [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+showplot = True
+outputtxt = True
+
+# hr        release are vertical stretch
+# vol       volume of snow in release area
+# dh        release snow thickness
+# xStart    upper margin of release area, x-coordinate
+# lenP      number of polygon points
+# showplot  show plot of release area
+# outputtxt copy the output to txt file to be read in QGIS
```
Comment (fso42) [inline avaframe/in3Utils/getReleaseAreaCfg.ini:22]: Make it consistent with other ini files and put the comments above each line...

File: avaframe/in3Utils/generateTopo.py [hunk 4]
```diff
@@ -460,18 +459,24 @@ def writeDEM(z, name_ext, ncols, nrows, dx, cfgDEM):
...
-    if os.path.isfile('avaframe/in3Utils/local_generateTopoCfg.ini'):
-        cfg.read('avaframe/in3Utils/local_generateTopoCfg.ini')
+    outDir = os.path.join(avalancheDir, 'Inputs')
+    if os.path.isdir(outDir):
+        log.warning('Be careful %s already existed - new data is added' % (outDir))
     else:
-        cfg.read('avaframe/in3Utils/generateTopoCfg.ini')
+        os.makedirs(outDir)
+
+    return outDir
```
[the remark this answers is not in the document]
Comment (awirb) [reply avaframe/in3Utils/generateTopo.py:474]: I adjusted the code to directly set the outDir and if NameOfAvalanche/Inputs does not exist, it throws an error

File: avaframe/in3Utils/getReleaseArea.py [hunk 5]
```diff
@@ -0,0 +1,247 @@
...
+    xyPoints[2, 0] = xStop             # xur
+    xyPoints[2, 1] = 0.5 * yr           # yur
+    xyPoints[3, 0] = xStop - xrp       # xul
+    xyPoints[3, 1] = 0.5 * yr           # yul
+
+    # list all points
+    for m in range(lenp):
+        log.info('Point %d: x %f y %f' % (m, xyPoints[m, 0], xyPoints[m, 1]))
+
+    return xv, yv, xyPoints
```
[the remark this answers is not in the document]
Comment (awirb) [reply avaframe/in3Utils/getReleaseArea.py:128]: I made it a bit shorter by introducing a makexyPoints function and directly give xv, yv

File: avaframe/in3Utils/getReleaseArea.py (hunk 1 above)
Comment (awirb) [reply avaframe/in3Utils/getReleaseArea.py:161]: ah is now changed to d0

File: avaframe/in3Utils/getReleaseArea.py (hunk 2 above)
Comment (awirb) [reply avaframe/in3Utils/getReleaseArea.py:183]: Now outDir is set directly

File: avaframe/in3Utils/getReleaseAreaCfg.ini (hunk 3 above)
Comment (awirb) [reply avaframe/in3Utils/getReleaseAreaCfg.ini:22]: adjusted

**Pull request merged**